csrankings.csv

```diff
@@ -3361,7 +3365,7 @@ Dirk Beyer 0001,LMU Munich,https://www.sosy-lab.org/people/beyer,AFOIl1IAAAAJ
 Dirk Englund,Massachusetts Institute of Technology,http://www.rle.mit.edu/qp,ZFpENKoAAAAJ
 Dirk Fahland,TU Eindhoven,https://www.win.tue.nl/~dfahland,qLkK5JAAAAAJ
 Dirk Grunwald,University of Colorado Boulder,http://systems.cs.colorado.edu/people/faculty/dirk-grunwald,3Ws6G2AAAAAJ
-Dirk Hovy,University of Copenhagen,http://www.dirkhovy.com,7xluaTAAAAAJ
+Dirk Hovy,Bocconi University,http://www.dirkhovy.com,7xluaTAAAAAJ

```

Comment:
Bocconi is not in the database, so this entry should be removed.

Reply:
rm-ed